### DISCUSSION TITLE
Bulk products

### DISCUSSION THREAD

**Discussion by pavelekNET** [vendurehq/vendure#567] [General]
It would be great to create bulk product where I can "config" what products the whole bulk will contain and if its required, free, discounted or optional.

It should work for example like this (box protector is optional item):

![image](https://user-images.githubusercontent.com/8962160/99738751-83a09900-2acb-11eb-9133-a8e6f4db0c74.png)

**Comment by michaelbromley** [comment]
Hi, there is a related feature request issue here: https://github.com/vendure-ecommerce/vendure/issues/236

I believe this functionality could be implemented as a plugin already, but I agree it would be good to have available either as part of the core, or as an official plugin.

**Reply by pavelekNET** [reply]
Hi Michael,

where I can find some list of official plugins? What I am missing next is "preorders" - i know that there are backorders already but preorder management is more complicated. I can post more info about preorders if needed.

**Reply by michaelbromley** [reply]
The only list of plugins right now is here: https://www.vendure.io/docs/plugins/available-plugins/

There is a community effort to gather a list of all other plugins but it's not done yet. 

I would be happy to hear more about pre-orders, please describe how it should work.
